build/vars.build.mk

```diff
@@ -14,15 +13,14 @@ LOCAL_AM_CFLAGS = \
 	-DDOCDIR=\""$(docdir)"\" \
 	-DGEANYPLUGINS_DATADIR=\""$(datadir)"\" \
 	-DPKGDATADIR=\""$(pkgdatadir)"\" \
-	-DLIBDIR=\""$(libdir)"\" \
-	-DPKGLIBDIR=\""$(pkglibdir)"\"
```

Comment:
ok, not used for the moment.  FWIW, would be `"lib/$(PACKAGE)"` on Windows I guess -- or simply `"lib"` as it's already likely to be package-specific